cli/src/pcluster/models/imagebuilder.py

```diff
@@ -173,15 +178,21 @@ def __init__(
         self._add_validator(
             EbsVolumeTypeSizeValidator, priority=10, volume_type=Param("gp2"), volume_size=self.image.root_volume.size
         )
+        self._add_validator(
+            EBSAMIVolumeSizeValidator,
+            priority=9,
+            volume_size=self.image.root_volume.size,
+            image=self.build.parent_image,
+        )
 
     def _set_default(self):
         # set default root volume
         if self.image.root_volume is None or self.image.root_volume.size.value is None:
-            increase_volume_size = 15
+            reserved_size = 15
             ami_id = utils.get_ami_id(self.build.parent_image.value)
```

Comment:
not strictly related to this patch but by looking at this get_ami_id method I have some comments:
1. It's  ImageBuilder specific, in fact it's using ImageBuilder boto3 client. I'll keep it in the imagebuilder.py model or in a imagebuilder_utils.py to avoid that someone start using this utility method for the base cli (we don't want it because permission for calling ImageBuilder are only required for the createami command).
2. Let's rename to evaluate_ami_id(ami) to be clear that the method is evaluating the id from the given param.
3. Let's rename the parameter to ami, rather than parent_image that is too strictly related to the image builder stuff.

Reply:
I'd like to rename the function name to `evaluate_ami_id(ami)` and rename parameter to ami to make it more generic, and I still keep this in utils, because if we need to evaluate_ami_id by other info (e.g. ami name) in future, we can extend this function.